pkg/serializer/jsonstream/compressor.go

```diff
@@ -132,7 +134,7 @@ func (c *compressor) pack() error {
 func (c *compressor) addItem(data []byte) error {
 	// check item size sanity
 	if !c.checkItemSize(data) {
-		return errTooBig
+		return ErrTooBig
```

Comment:
Could you add tests that this logic works well ? This is very unlikely to have ever been triggered on metrics or service checks

Reply:
`TestSendV1EventsCreateMarshalerCollection` actually tests this code as part of the logic of `serializeEventsStreamJSONMarshalerPayload` (Checked with the debugger).